### DISCUSSION TITLE
Alias and Function config files

### DISCUSSION THREAD

**Discussion by trymeouteh** [nushell/nupm#99] [Ideas]
I would like to suggest having Nu generate a `alias.nu` and `functions.nu` files when the user first run Nu and will decide where these scripts are stored.

These two scripts will be empty and can be used as a place for the user to add custom aliases and functions which can be accessed in the shell. Kinda like adding aliases and functions to `.bashrc` file for Bash but for Nu and having separate files to keep the custom aliases and functions in a separate file.

**Comment by amtoine** [comment]
@trymeouteh :wave:

that's an idea, but is it related to Nupm? :confused:

**Reply by trymeouteh** [reply]
https://github.com/nushell/nushell/discussions/13473

**Reply by amtoine** [reply]
thanks :pray:
